x-pack/plugins/security_solution/public/common/components/links/helpers.tsx

```diff
@@ -24,6 +24,12 @@ export const LinkAnchor: React.FC<EuiLinkProps> = ({ children, ...props }) => (
   <EuiLink {...props}>{children}</EuiLink>
 );
 
+export const SpacedLinkAnchor = styled(LinkAnchor)`
+  margin-right: 5px;
+`;
```

Comment:
It was done to avoid links to be crowded when they rendered as siblings

w/ spacing
<img width="708" alt="Screenshot 2021-11-22 at 17 22 54" src="https://user-images.githubusercontent.com/92328789/142907041-8762a74a-cbba-4f2e-83c6-910f1dfab1aa.png">

w/o spacing
<img width="693" alt="Screenshot 2021-11-22 at 17 24 32" src="https://user-images.githubusercontent.com/92328789/142907092-3236f10b-a463-47b1-afc7-bb0bb862f01b.png">

I've introduced similar to Spacer component - RightSideSpacer to handle this use case

Reply:
++ @xcrzx here. This has been a common theme around the codebase for some time, as it can sometimes be tricky to get spaces to show up when iterating lists in react. The `Host Details` page does something similar with a `Spacer` component as well here:

https://github.com/elastic/kibana/blob/35c83bac5a22fe79e8b9cafa1fd99817cdc36552/x-pack/plugins/security_solution/public/timelines/components/field_renderers/field_renderers.tsx#L211-L223

That said, if it's possible to get it to work using actual characters `,[space]` that'll be preferred as it scales when users change font-sizes or zoom.

Reply:
refactored to use characters.
Although, had to do something like this:

```typescript
      <>
            <EuiToolTip>
              <>
                <LinkAnchor>
                  {name}
                  {isSeparator && ','}
                </LinkAnchor>
              </>
            </EuiToolTip>
            {isSeparator && ' '}
          </>

```

I'ts because EuiToolTip has to be inline-block and spaces are not working within it. And if I put comma outside tooltip, I've started to get align issues + it will not be the same color as link.